Directory.Build.props

```diff
@@ -3,31 +3,11 @@
 <Project>
 
   <PropertyGroup>
-    <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
```

Comment:
This is incorrect, Directory.Build.props are added to MSBuildAllProjects by the importing file, they do not add themselves.

Reply:
@tannergooding I think all .props and .targets should add themselves to MSBuildAllProjects.  It doesn't hurt to have the same file added twice, and it's much simpler to always do it rather than have to reason about whether the importer is doing it for you.